go.mod

```diff
@@ -67,7 +64,7 @@ replace (
 	golang.org/x/lint => golang.org/x/lint v0.0.0-20201208152925-83fdc39ff7b5 // indirect
 	golang.org/x/oauth2 => golang.org/x/oauth2 v0.0.0-20210112200429-01de73cf58bd
 	golang.org/x/sync => golang.org/x/sync v0.0.0-20201207232520-09787c993a3a // indirect
-	google.golang.org/api => google.golang.org/api v0.30.0
+	google.golang.org/api => google.golang.org/api v0.46.0
 	google.golang.org/genproto => google.golang.org/genproto v0.0.0-20210111234610-22ae2b108f89 // indirect
 	google.golang.org/grpc => google.golang.org/grpc v1.27.1 // indirect
 	gopkg.in/check.v1 => gopkg.in/check.v1 v1.0.0-20200902074654-038fdea0a05b // indirect
```

Comment:
doesn't this have to pick up a compatible version of `k8s.io/legacy-cloud-providers` that works properly with google.golang.org/api v0.46.0?

Reply:
@jpbetz